llama/llama.go

```diff
@@ -357,6 +390,7 @@ type SamplingParams struct {
 	TfsZ           float32
 	TypicalP       float32
 	Temp           float32
+	RepeatLastN    int
```

Comment:
Note for later: I believe we're missing `min_p` as well

Reply:
Good catch, I’ll put that on the list for follow up.